backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Core/Services/ResourceService.cs

```diff
@@ -147,29 +154,68 @@ public async Task<ServiceResource> GetResource(string resourceRegistryId)
 
             if (!_memoryCache.TryGetValue(cacheKey, out ServiceResource resource))
             {
-                resource = await _resourceRegistryClient.GetResource(resourceRegistryId);
+                resource = await _resourceClient.GetResource(resourceRegistryId);
 
-                var cacheEntryOptions = new MemoryCacheEntryOptions()
-               .SetPriority(CacheItemPriority.High)
-               .SetAbsoluteExpiration(new TimeSpan(0, _cacheConfig.ResourceRegistryResourceCacheTimeout, 0));
+                MemoryCacheEntryOptions cacheEntryOptions = new MemoryCacheEntryOptions()
+                    .SetPriority(CacheItemPriority.High)
+                    .SetAbsoluteExpiration(new TimeSpan(0, _cacheConfig.ResourceRegistryResourceCacheTimeout, 0));
 
                 _memoryCache.Set(cacheKey, resource, cacheEntryOptions);
             }
 
             return resource;
         }
 
+        /// <inheritdoc />
+        public async Task<List<ResourceOwnerFE>> GetAllResourceOwners(string languageCode)
+        {
+            OrgList orgList = new OrgList();
+            try
+            {
+                orgList = await _resourceClient.GetAllResourceOwners();
+            }
+            catch (Exception ex)
+            {
+                _logger.LogError("//ResourceService //GetAllResourceOwners failed exception: {Ex}", ex);
+            }
+
+            return MapOrgListToResourceOwnerFe(orgList, languageCode);
+        }
+
+        /// <inheritdoc />
+        public List<ResourceOwnerFE> MapOrgListToResourceOwnerFe(OrgList orgList, string languageCode)
```

Comment:
```suggestion
        private List<ResourceOwnerFE> MapOrgListToResourceOwnerFe(OrgList orgList, string languageCode)
```
Helper functions should be private

Reply:
Nice spotted

Reply:
It's used in ResourceControllerTest and should be public